plugins/modules/azure_rm_arcssh.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2024 Bill Peck  <bpeck@redhat.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import annotations
+
+DOCUMENTATION = '''
+---
+module: azure_rm_arcssh
+version_added: "3.0.0"
+short_description: Configure SSH proxy for ARC hosts
+description:
+    - Configure SSH proxy for ARC hosts. Use this in combination with the
+      azure_rm inventory plugin with include_arc_resource_groups to dynamically
+      generate the list of hosts.
+
+options:
+    resource_group:
+        description:
+            - The name of the Resource Group to which the server belongs.
+        required: True
+        type: str
+    resource_type:
+        description:
+            - The name of the Resource Type to which the server belongs.
+        required: True
+        type: str
+    inventory_hostname:
+        description:
+            - The inventory configured hostname.
+        type: str
+    ansible_host:
+        description:
+            - The discovered hostname that is generated by the inventory config.
+            - Please see the examples on how this can be set based on the resource_type.
+        type: str
+    private_key_file:
+        description:
+            - The Private key that matches the public key that you have already added to your authorized_hosts
+              of local_user.
+        default: ~/.ssh/id_rsa
+        type: str
+    local_user:
+        description:
+            - The local user that you use to log into your server.
+        type: str
+    port:
+        description:
+            - The port that your ssh server is listening on.
+        default: 22
```

Comment:
The defined type is 'str', should it be like this' 22 '；
Or define its type as int.
```suggestion
        default: ‘22’
```